appear/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project>
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.arcbees.gquery</groupId>
+        <artifactId>appear-plugin</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <name>Appear plugin for gQuery</name>
+    <artifactId>appear</artifactId>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
+                <includes>
+                    <include>**/*.java</include>
+                    <include>**/*.gwt.xml</include>
+                </includes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+            </resource>
+        </resources>
+
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+            </plugin>
```

Comment:
Unless it goes in GWT-Query or GWT, we're sticking to Java conventions for Arcbees code base